bin/disco_purge_snapshots.py

```diff
@@ -55,7 +58,8 @@ def run():
     # If no options are set, we assume user wants all of 'em.
     arg_options = ["--stray-ami", "--no-metadata",
                    "--keep-days", OLD_IMAGE_DAYS,
-                   "--keep-num", DEFAULT_KEEP_LAST]
+                   "--keep-num", DEFAULT_KEEP_LAST,
+                   "--max-per-day", None]
```

Comment:
Since `--keep-num` is evaluated after `--max-per-day`, would it make sense to swap the order of these two arguments?